notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,928 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "85bb873e-562a-4d60-9536-b6b88aecb5c2",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Working with GPUs in OpenVINO™"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6c8277d-29b3-4172-8600-fcbbbbdbafd2",
+   "metadata": {},
+   "source": [
+    "This tutorial provides a high-level overview of working with Intel GPUs in OpenVINO. It shows users how to use Query Device to list system GPUs and check their properties, and it explains some of the key properties. It shows how to compile a model on GPU with performance hints and how to use multiple GPUs using MULTI or CUMULATIVE_THROUGHPUT. \n",
+    "\n",
+    "The tutorial also shows example commands for benchmark_app that users can run to compare GPU performance in different configurations. It also provides code for a basic end-to-end application that compiles a model on GPU and uses it to run inference."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e7253ed-d8a1-4475-8e83-263336639157",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab1f9f13-63a4-4b2d-95e3-9a82c1e9cebd",
+   "metadata": {},
+   "source": [
+    "Originally, graphic processing units (GPUs) began as specialized chips developed to accelerate the rendering of computer graphics. In contrast to CPUs, which have few but powerful cores, GPUs have many more specialized cores, making them ideal for workloads that can be parallelized into simpler tasks. Nowadays, one such workload is deep learning, where GPUs can easily accelerate inference of neural networks by splitting operations across multiple cores.\n",
+    "\n",
+    "OpenVINO supports inference on Intel integrated GPUs (which are included with most Intel processors) or on Intel discrete GPU products like the [Intel Arc™ A-Series graphics cards](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/arc.html). To get started, first [install OpenVINO](https://docs.openvino.ai/latest/openvino_docs_install_guides_overview.html) on a system equipped with one or more Intel GPUs. Follow the [GPU configuration instructions](https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gpu.html) to configure OpenVINO to work with your GPU. Then, read on to learn how to accelerate inference with GPUs in OpenVINO!"
```

Comment:
```suggestion
    "OpenVINO supports inference on Intel integrated GPUs (which are included with most [Intel® Core™ desktop and mobile processors](https://www.intel.com/content/www/us/en/products/details/processors/core.html)) or on Intel discrete GPU products like the [Intel® Arc™ A-Series Graphics cards](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/arc.html) and [Intel® Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/flex-series.html). To get started, first [install OpenVINO](https://docs.openvino.ai/latest/openvino_docs_install_guides_overview.html) on a system equipped with one or more Intel GPUs. Follow the [GPU configuration instructions](https://docs.openvino.ai/latest/openvino_docs_install_guides_configurations_for_intel_gpu.html) to configure OpenVINO to work with your GPU. Then, read on to learn how to accelerate inference with GPUs in OpenVINO!"
```

Reply:
When using full product names we are required to include (R) and (TM). I updated to also include Data Center Flex GPU (codenamed Arctic Sound) and clarified that iGPU are included in most Core desktop and mobile processors (Xeon processors do not include iGPU).

Reply:
Great, thanks Ryan! 

Now that you mention it (we forgot to add it in the PR description), how should we handle the (R) and (TM) symbols across the notebook? Should we always use them each time Intel/OpenVINO is mentioned or just on product mentions like you did?